.github/workflows/forge-pfn.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
         id: determine-test-branch
         run: |
           if [[ "${{ github.event_name }}" == "schedule" ]]; then
-            if [[ "${{ github.event.schedule }}" == "0 8 */2 * *" ]]; then
+            if [[ "${{ github.event.schedule }}" == "0 3 */2 * *" ]]; then
```

Comment:
Somebody can miss changing this easily 😄

Reply:
Yeah, it's a bit annoying 😄 I've done it at least once. We should probably refactor all of these files to have a single cron string defined, or just get rid of the check altogether?